### PR TITLE
source: devices: AM62*: linux: Updated LPM power measurements for SDK 10.0

### PR DESCRIPTION
**Pull request by kwillis01**
Updated DeepSleep and MCU Only LPM power measurements in the Linux performance guide.

### CODE REVIEW

**Comment by kwillis01** [comment]
@madweshanshu

**Comment by cshilwant** [comment]
@kwillis01 What's the reason of updating SDK 10.0 power numbers now?
This eventually won't get published on software-dl.ti.com and with SDK 10.1 around the corners it might even get updated with revised numbers.

File: source/devices/AM62PX/linux/Linux_Performance_Guide.rst [hunk 1]
```diff
@@ -1088,26 +1088,26 @@ Table:  **Deep sleep**
 .. csv-table::
     :header: "Rail name","Rail voltage(V)","am62pxx_sk-fs"
```
Comment (nmenon) [inline source/devices/AM62PX/linux/Linux_Performance_Guide.rst:1089]: please fix units of "am62pxx_sk-fs" - is that Watts? Watt hour? milliWatts? uWatts?

**Comment by kwillis01** [comment]
@cshilwant Is there anyway software-dl.ti.com can be updated with the 10.0 numbers sooner. Currently SDK 9.2 power measurements is in the release notes for 10.0.

File: source/devices/AM62PX/linux/Linux_Performance_Guide.rst (hunk 1 above)
Comment (kwillis01) [reply source/devices/AM62PX/linux/Linux_Performance_Guide.rst]: fixed to be uW for AM62x, p, and a

**Comment by cshilwant** [comment]
@kwillis01 In general, the plan was to stop SDK 10.0 documentation syncing once the SDK 10.1 RCs started. The merge window for documentation bug fixes for SDK 10.1 is now open. So, doing a documentation syncing now might lead syncing of any SDK 10.1 specific commit in SDK 10.0 docs (Will need to see if there any PRs that went in)

The suggestion would be to wait until SDK 10.1 & fix the power numbers on 10.1 baseline.

**Review by cshilwant — APPROVED** [review]

**Comment by cshilwant** [comment]
@kwillis01 The permission error observed with Commit Check workflow is now fixed.
Can you rebase your PR on top of latest master?

**Review by cshilwant — CHANGES_REQUESTED** [review]
Waiting for PR to be rebased with master

**Comment by cshilwant** [comment]
@kwillis01 The workflow throws an error. Please check https://github.com/TexasInstruments/processor-sdk-doc/actions/runs/11631469679#summary-32394326587 for more details on what's needed to be fixed in commit message

Refer https://github.com/TexasInstruments/processor-sdk-doc/blob/master/CONTRIBUTING.md for more details on CONTRIBUTING guidelines

**Review by DhruvaG2000 — APPROVED** [review]
@kwillis01  For commit style, see for example https://github.com/TexasInstruments/processor-sdk-doc/commit/cf40654e4125f0fe1aacc6975f7881396f41a2d8 . This is not the same way we do commits for linux kernel. @cshilwant  has shared an useful link to contrib guide as well that you can refer to.
With that addressed, the changes in the docs look good. Thanks for fixing the am62a/dev names in the table to power/mw.

**Review by praneethbajjuri — APPROVED** [review]

**Review by cshilwant — APPROVED** [review]

**Pull request merged**